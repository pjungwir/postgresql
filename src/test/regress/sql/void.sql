select void_send('foo');
