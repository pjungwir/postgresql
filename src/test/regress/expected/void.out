select void_send('foo');
 void_send 
-----------
 \x
(1 row)

